### ISSUE TITLE
The application is not open anymore issue in mac

### ISSUE THREAD

**Issue by Aditya1349** [grishka/NearDrop#162]
Whenever i am sending any files from my Android to the mac, and when the mac gets the connection request and when i click on that notification to accept it. It says "The app "NearDrop" isn't open anymore" and this is happening every time with me. But let me tell you the app NearDrop is always open on mac. I think it needs an update i guess.
<img width="1440" alt="Screenshot 2024-05-21 at 1 46 30 PM" src="https://github.com/grishka/NearDrop/assets/137182238/faf67a91-412c-4a39-91c4-855ca049b542">

**Comment by steffandroid** [comment]
Also getting this, but only after I changed NearDrop's notifications to "alerts" in the system settings per https://github.com/grishka/NearDrop/issues/161#issuecomment-2115642280.

**Comment by Aditya1349** [comment]
So after setting the notification to "alert" it worked ?... because mine is not working still.

**Comment by steffandroid** [comment]
No, the notifications disappeared quickly when set to "banner", but I didn't see "The app "NearDrop" isn't open anymore". After switching to "alert" I now see that error.

**Comment by medoevruslan** [comment]
have the same issue
Running macOS 14.4.1

**Comment by steffandroid** [comment]
This is now fixed in [2.0.3](https://github.com/grishka/NearDrop/releases/tag/v2.0.3).

**Comment by DayDreamingTime** [comment]
Works perfectly now.

**Comment by pietschy** [comment]
Working again now for me too.  Thanks heaps 👍

**Comment by grishka** [comment]
@xa4hf8 please stop commenting on everything you see. You aren't being helpful.

**Comment by steffandroid** [comment]
@yairchu bit hard to say, could be https://github.com/grishka/NearDrop/commit/10d63f13bde485a1c5c35186eba498117b7e7230?

**Comment by yairchu** [comment]
@steffandroid: Thanks! In the end I found the problem in my app was with it taking too long before setting up the macOS menu. Doesn't seem same thing here maybe it was a similar symptom for different reason.
